v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/loadtesting/postgresql/SourceDbToSpannerLT.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.loadtesting.postgresql;
+
+import com.google.cloud.teleport.metadata.TemplateLoadTest;
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.iowrapper.config.SQLDialect;
+import com.google.cloud.teleport.v2.templates.DataStreamToSpanner;
+import com.google.cloud.teleport.v2.templates.loadtesting.SourceDbToSpannerLTBase;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@Category(TemplateLoadTest.class)
+@TemplateLoadTest(DataStreamToSpanner.class)
+@RunWith(JUnit4.class)
+public class SourceDbToSpannerLT extends SourceDbToSpannerLTBase {
+
+  @Test
+  public void backfill100Gb() throws IOException, ParseException, InterruptedException {
+    String username =
+        accessSecret(
+            "projects/269744978479/secrets/nokill-sourcedb-postgresql-to-spanner-cloudsql-username/versions/1");
+    String password =
+        accessSecret(
+            "projects/269744978479/secrets/nokill-sourcedb-postgresql-to-spanner-cloudsql-password/versions/1");
+    String database =
+        accessSecret(
+            "projects/269744978479/secrets/nokill-sourcedb-postgresql-to-spanner-cloudsql-database/versions/1");
+    String host =
+        accessSecret(
+            "projects/269744978479/secrets/nokill-sourcedb-postgresql-to-spanner-cloudsql-ip-address/versions/1");
+    int port =
+        Integer.parseInt(
+            accessSecret(
+                "projects/269744978479/secrets/nokill-sourcedb-postgresql-to-spanner-cloudsql-port/versions/1"));
+
+    setUp(SQLDialect.POSTGRESQL, host, port, username, password, database);
+    createSpannerDDL("SourceDbToSpannerLT/postgresql/spanner-schema.sql");
```

Comment:
nit: should this be part of the setUp itself?

Reply:
I wanted to leave it out of the setup so that the schema is configurable for other load tests. For instance, we could have another test that connects to a different PG instance and then creates a different Spanner schema to be validated.

We could have added the resource to the setUp call, but I believe this is more flexible.